docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/event-card/event-card.stories.js

```diff
@@ -34,7 +34,7 @@ export const EventCard = (knobTab) => {
     date: date('Date', new Date(), generalKnobTab),
     title: text('Title', 'Event name which runs across two or three lines', generalKnobTab),
     location: text('Location', 'Suburb, State – 16:00–17:00', generalKnobTab),
-    summary: text('Summary', 'Card summary using body copy which can run across multiple lines. Recommend limiting this summary to three or four lines..', generalKnobTab),
+    summary: text('Summary', randomSentence(), generalKnobTab),
```

Comment:
here and below using random sentences in summaries to test card heights in the container